core/tests/puppeteer-acceptance-tests/spec/translation-admin-tests/provide-translation-rights.spec.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for translation admins to provide translation
+ * rights from users.
+ */
+
+const userFactory = require(
+  '../../puppeteer-testing-utilities/user-factory.js');
+const testConstants = require(
+  '../../puppeteer-testing-utilities/test-constants.js');
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Translation Admin', function() {
+  let translationAdmin = null;
+
+  beforeAll(async function() {
+    translationAdmin =
+      await userFactory.createNewTranslationAdmin('translationAdm');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should be able to provide translation rights to user.',
+    async function() {
+      await userFactory.createNewGuestUser(
+        'translatorSpanish', 'translatorSpanish@example.com');
+      await translationAdmin.navigateToContributorDashboardAdminPage();
```

Comment:
you can simply use `goto` function within the user-utility file with the just below function `assignTranslationRights`

Reply:
Isn't this way consistent with how the [blog post admin utility file](https://github.com/oppia/oppia/blob/develop/core/tests/puppeteer-acceptance-tests/user-utilities/blog-post-admin-utils.js#L62) navigates?

Reply:
`navigateTo` can be more generalized, if you are naviagting multiple times in other tests too, I would recommend to use the more generalized function otherwise it is okay!

Reply:
Sounds good.